### PR TITLE
Feature/update to polkadot v0 9 9

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -199,7 +198,7 @@ parameter_types! {
 const_assert!(NORMAL_DISPATCH_RATIO.deconstruct() >= AVERAGE_ON_INITIALIZE_RATIO.deconstruct());
 
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = ();
+	type BaseCallFilter = Everything;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:201]: As far I understand it it is a refactoring. Now we need to explicitly say that all calls to the runtime from SC are allowed.
Here is an [original PR](https://github.com/paritytech/substrate/pull/9276)

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -348,6 +351,7 @@ impl pallet_babe::Config for Runtime {
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
 	type EpochChangeTrigger = pallet_babe::ExternalTrigger;
+	type DisabledValidators = Session;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:354]: A [fix](https://github.com/paritytech/substrate/pull/9414) which doesn't allow disabled validators to build blocks.

File: client/executor/Cargo.toml [hunk 3]
```diff
@@ -47,7 +47,6 @@ sp-state-machine = { version = "0.10.0-dev", path = "../../primitives/state-mach
 sp-runtime = { version = "4.0.0-dev", path = "../../primitives/runtime" }
 sp-tracing = { version = "4.0.0-dev", path = "../../primitives/tracing" }
 sp-maybe-compressed-blob = { version = "4.0.0-dev", path = "../../primitives/maybe-compressed-blob" }
-sp-tracing = { version = "3.0.0", path = "../../primitives/tracing" }
 tracing = "0.1.25"
```
Comment (AndreiNavoichyk) [inline client/executor/Cargo.toml:51]: Why we don't need it?

File: primitives/api/src/lib.rs [hunk 4]
```diff
@@ -490,12 +490,6 @@ pub trait ApiExt<Block: BlockT> {
 	where
 		Self: Sized;
 
-	/// Returns the version of the given api.
-	fn api_version<A: RuntimeApiInfo + ?Sized>(
```
Comment (AndreiNavoichyk) [inline primitives/api/src/lib.rs:485]: Why we don't need it?

File: bin/node/runtime/src/lib.rs [hunk 5]
```diff
@@ -558,9 +563,23 @@ sp_npos_elections::generate_solution_type!(
 pub const MAX_NOMINATIONS: u32 =
 	<NposCompactSolution16 as sp_npos_elections::CompactSolution>::LIMIT as u32;
 
+/// The numbers configured here should always be more than the the maximum limits of staking pallet
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:566]: Does it correct for us?

File: client/executor/Cargo.toml (hunk 3 above)
Comment (MRamanenkau) [reply client/executor/Cargo.toml:51]: Are you asking about `tracing = "0.1.25"`? It's not removed.
If you asking about `sc-tracing`, in not removed. Please, check the final version.

File: primitives/api/src/lib.rs (hunk 4 above)
Comment (MRamanenkau) [reply primitives/api/src/lib.rs:485]: It's a dublicate left after merge. Check line 485

File: bin/node/runtime/src/lib.rs (hunk 5 above)
Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs:566]: Yes, our values are orders of magnitude less.
[This discussion](https://github.com/paritytech/substrate/pull/9286#discussion_r666421141) might make it a bit cleaner

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**